src/App.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Calculator from './components/Calculator';
+
+function App() {
+  return (
+    <div className="App">
+      <Calculator />
+    </div>
+  );
+}
```

Comment:
- [ ] As requested by the previous reviewer, Please change this component to a class-based component to follow the project requirements.
![image](https://user-images.githubusercontent.com/83803414/183135624-a7f49e74-dee5-4de6-91fb-5eb6968b33d4.png)